zkevm-circuits/src/evm_circuit/execution/sstore.rs

```diff
@@ -1129,4 +1142,28 @@ mod test {
             false,
         );
     }
+
+    #[test]
+    fn sstore_simple_with_busmapping() {
+        let bytecode = bytecode! {
+            #[start]
+            PUSH32(0x030201) // value 
+            PUSH32(0x060504) // key
+            SSTORE
+            PUSH32(0x030202)
+            SLOAD
+            STOP
+        };
+        /*
+            SSTORE
+            PUSH32(0x030201)
+            SLOAD */
+
+
```

Comment:
shouldn't this be put in zkevm-circuits/src/evm_circuit/execution/sstore.rs?

and then we
```
PUSH32(0x030201) // value 
PUSH32(0x060504) // key
SSTORE
#[start] // !!!start here
PUSH32(0x030202)
SLOAD
STOP
```

Reply:
Oh I see. there's no bus_mapping::sstrore yet. That's why you code won't fail ATM.

Reply:
The pr is only draft. So i just put it random location.    While sstore bus mapping has been implemented